IOS Practicals New/ViewControllers/LogInViewController.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+class LogInViewController: UIViewController {
+    
+    //MARK: - Outlets
+    @IBOutlet weak var lblSignUp: UILabel!
+    
+    //MARK: - UI View Controller
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let tap = UITapGestureRecognizer(target: self, action: #selector(lblSignUpClick))
+        lblSignUp.isUserInteractionEnabled = true
+        lblSignUp.addGestureRecognizer(tap)
+    }
+}//End of LogInViewController
```

Comment:
Put one blank line space before ending class bracket .
and you need to mark as // End of class